samples/BuilderApp/BuilderApp.csproj

```
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworks>net472;netcoreapp2.1;net5.0</TargetFrameworks>
+    <TargetFrameworks>net46;netcoreapp3.1;net5.0</TargetFrameworks>
```

No need to back these down IMO, we only really care about the main library's TFs.
```suggestion
    <TargetFrameworks>net472;netcoreapp3.1;net5.0</TargetFrameworks>
```

---

I was thinking it seems weird to test something that's at all different from the real product. Don't think it matters too much, though.